Update iOS share URLs for SSATR (PLAYRTS-5486) (#471)
SRGSSR/playsrg-apple: docs/REMOTE_CONFIGURATION.md

* `songsViewStyle` (optional, string): The songs view style when added to the view. Never displayed if not set. Available values are:
   * `collapsed`: Collapsed when added to the view.
   * `expanded`: Expanded when added to the view.

> * `shareURL` (optional, string): The URL used to share the livestream of a channel.

Reviewer: ```suggestion
* `shareURL` (optional, string): The URL used to share the channel website.
```
